src/main/resources/com/ncc/aif/restricted_hypothesis_aif.shacl

```diff
@@ -24,3 +24,32 @@ aida:SystemShape
             HAVING ((COUNT(?x)) != 1)
         """ ;
     ] .
+
+########################
+# 4.3 TA3 #7 Each entity (cluster) in the hypothesis graph must
+# have exactly one text description (called a "handle")
+#------------------------
+aida:HypothesisEntityClusterRequireHandle
+    a sh:NodeShape ;
+
+    sh:target [
+      a sh:SPARQLTarget ;
+      sh:select """
+            PREFIX aida:  <https://tac.nist.gov/tracks/SM-KBP/2018/ontologies/InterchangeOntology#>
+            SELECT ?entityCluster
+            WHERE {
+                ?entityCluster a aida:SameAsCluster .
+                ?entityCluster aida:prototype ?prototype .
+                ?prototype a aida:Entity .
```

Comment:
This and the line above can be replaced with:
`?entityCluster aida:prototype/a aida:Entity .`